cucumber/policy/features/replace.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Replacing a policy
+
+A policy can be reloaded using the --replace flag
+
+  Scenario: A multifile policy with one modified file fails on reload
+
+    Given I replace the policy by loading the policy file "multifile1.yml"
+    And I load a "root" policy file "multifile2.yml"
+    When I replace the policy by loading the policy file "multifile2_with_deletes.yml"
+    Then there is an error
+    And the error code is "not_found"
+    And the error message is "ID cucumber:group:developers is not a valid role"
```

Comment:
This message makes it sound a little bit like `cucumber:group:developers` is not a valid role id. I think it would be more clear to say that the role does not exist.

Reply:
So updated